index.js

```diff
@@ -209,16 +209,23 @@ Practice accessing data above by console.log-ing following items:
 
 //(1) Name of the first artist (0th index) in the array
 
-
+console.log(artists[0].name)
 
 //(2) Bio of the third artist (2nd index) in the array 
 
+console.log(artists[2].bio)
 
 
 /* 🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀 Task 2 (not auto tested): 🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀 
 (no function needed) 
 There is a typo in your dataset 😱 The 9th artist, Vincent Van Gogh is currently Vincent Van Dough. Fix this issue and console.log() to check your work. */
 
+const changeArtist = function(array, index, incorrectKey, correctName){
+  array[index][incorrectKey] = correctName;
+  return array
+}
```

Comment:
Awesome job of making a reusable function. The instructions said you didn't need a function but definitely an awesome thing if you need to use it again.